### ISSUE TITLE
Accessibility to 100%

### ISSUE THREAD

**Issue by dietza** [JeffKersting/refactor-tractor-whats-cookin#2]
As a developer,

When I am building an app,

I want to incorporate accessibility considerations early on in the build,

So that the application is easily usable by as many people as possible despite differences in experiences and abilities, and reaches this point efficiently.

- [ ] You must be able to tab through your app and use it without a mouse
- [x] Your app must still be usuable when tested with a colorblind extension
- [x] score as close to 100% as possible with the “Lighthouse Accessibility Audit”. Be prepared to explain any accessibility audits your application is failing.
- [ ] Your HTML must be written semantically and ARIA tags should be used (ONLY if needed / appropriate)

**Comment by dietza** [comment]
Some concerns in working on accessibility further into this project - I am finding that certain elements in HTML share id's, which should always be unique, and that class names are not always clear/helpful.
With this in mind, incorporating ARIA tags and improving the Lighthouse audit are made a little more tedious in checking cross-referenced files to be sure that resolving these basic points will not break functionality.

**Comment by srslie** [comment]
It's a good idea to be doing these as we go, maybe in the future we should have a checkbox in PRs for 'accessibility audit' for each PR?

**Comment by dietza** [comment]
I like that idea

**Comment by dietza** [comment]
Tab indexes are all set to '0' and will follow the order of elements as sequentially ordered in HTML - this could be done better to allow a User greater control over the flow of their experience, but ran into a series of unanticipated issues and multiple merge conflicts as multiple collaborators further manipulated these files while they were otherwise being updated for accessibility.
